libmc/_client.pyx

```diff
@@ -48,6 +48,17 @@ cdef extern from "Common.h" namespace "douban::mc":
         OPT_HASH_FNV1A_32
         OPT_HASH_CRC_32
 
+    ctypedef enum err_code_t:
+        SEND_ERR
+        RECV_ERR
+        CONN_POLL_ERR
+        POLL_TIMEOUT_ERR
+        POLL_ERR
```

Comment:
@zzl0 

```
/Users/kelvin/GitHub/douban/libmc/include/Common.h:186:3: error: expected identifier
  POLL_ERR = -4,
  ^
/usr/include/sys/signal.h:263:18: note: expanded from macro 'POLL_ERR'
#define POLL_ERR        4       /* [XSR] I/O error */
```

consider adding a unified prefix (e.g.; `MC_RETURN_`? ) to all enums of `err_code_t` type?